### DISCUSSION TITLE
[xstate/react] 1.0.0 released?

### DISCUSSION THREAD

**Discussion by cevr** [statelyai/xstate#1496] [General] [answered]
I see that xstate/react has been released, but I see no changelogs for the 1.0.0 version.

Is there a place we can find the changes made?

**Comment by davidkpiano** [comment, marked as answer]
The changelog is here: https://github.com/davidkpiano/xstate/blob/master/packages/xstate-react/CHANGELOG.md

The migration guide is here: https://xstate.js.org/docs/packages/xstate-react/#migration-from-0-x (there is not much to migrate).

I'm preparing a bigger post about this.
